src/Arcus.Templates.Tests.Integration/WebApi/WebApiProjectOptions.cs

```diff
@@ -221,7 +221,7 @@ private static string InsertInMemorySecretProviderCode(string startupContent, st
 
         private static string InsertSharedAccessAuthenticationHeaderSecretPair(string startupContent, string requestHeader, string secretName)
         {
-            return startupContent.Replace("YOUR REQUEST HEADER NAME", requestHeader)
+            return startupContent.Replace("X-Api-Key", requestHeader)
```

Comment:
Changed this because it may be a bit more user-friendly on start-up and you want to discover the API via the Swagger UI.

Reply:
Fine for me - Let's change it to `X-API-Key` then!